src/EditorFeatures/Core.Wpf/Suggestions/SuggestedActionsSource.cs

```diff
@@ -258,9 +282,9 @@ static SuggestedActionSetPriority ConvertToSuggestedActionSetPriority(UnifiedSug
                 // See https://github.com/dotnet/roslyn/issues/29589
                 const bool includeSuppressionFixes = true;
 
-                return UnifiedSuggestedActionsSource.GetFilterAndOrderCodeFixesAsync(
+                return await UnifiedSuggestedActionsSource.GetFilterAndOrderCodeFixesAsync(
```

Comment:
No await needed?

Reply:
simplified majorly (containing method is just removed).